Makefile.am

```diff
@@ -10,3 +13,39 @@ AM_DISTCHECK_CONFIGURE_FLAGS = --with-systemdsystemunitdir='$${libdir}/systemd/s
 pkgconfigdir = $(libdir)/pkgconfig
 
 pkgconfig_DATA = eos-updater-0.pc
+
+# Utility library
+noinst_LTLIBRARIES += libeos-updater-util/libeos-updater-util-@EUU_API_VERSION@.la
+
+libeos_updater_util_libeos_updater_util_@EUU_API_VERSION@_la_CFLAGS = \
+	$(WARN_CFLAGS) \
+	$(CODE_COVERAGE_CFLAGS) \
+	$(GIO_CFLAGS) \
+	$(SOUP_CFLAGS) \
+	$(OSTREE_CFLAGS) \
```

Comment:
Fair. I kept them in this order because they were copied verbatim from `src/Makefile.am`, but since enough of the other build rules changed in the transition, I can reorder this list. I will keep `WARN_CFLAGS` and `CODE_COVERAGE_CFLAGS` at the top, as it’s possible (though really really unlikely) that one of the dependency `_CFLAGS` variables would need to override them.